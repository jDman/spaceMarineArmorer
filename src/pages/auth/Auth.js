import React from 'react';

import classes from './Auth.module.css';

const Auth = (props) => {
  return <section className={classes.Auth}>{props.children}</section>;
};

export default Auth;
