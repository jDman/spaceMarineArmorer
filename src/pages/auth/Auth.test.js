import React from 'react';
import renderer from 'react-test-renderer';

import Auth from './Auth';

test('renders Auth component and children', () => {
  const component = renderer.create(
    <Auth>
      <p>Tested</p>
    </Auth>
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
