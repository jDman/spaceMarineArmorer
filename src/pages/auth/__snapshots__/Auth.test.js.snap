// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Auth /> renders Auth component and children 1`] = `
<section
  className="Auth"
>
  <p>
    Tested
  </p>
</section>
`;
